File: java/time.java
import java.time.ZonedDateTime;
import java.time.format.DateTimeFormatter;
import java.time.ZoneId;

public class TimeInCST {
    public static void main(String[] args) {
        ZoneId cstZone = ZoneId.of("America/Chicago");
        ZonedDateTime cstTime = ZonedDateTime.now(cstZone);
        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss z");
        String formattedTime = cstTime.format(formatter);
        System.out.println("Current time in CST: " + formattedTime);
    }
}

File: go/time.go
package main

import (
	"fmt"
	"time"
)

func main() {
	loc, err := time.LoadLocation("America/Chicago")
	if err != nil {
		fmt.Println("Error loading location:", err)
		return
	}
	cstTime := time.Now().In(loc)
	fmt.Println("Current CST time:", cstTime.Format("2006-01-02 15:04:05"))
}
